server/src/common/dto/notes.go

```diff
@@ -89,7 +88,9 @@ type NoteUpdateRequest struct {
 
 // NoteDeleteRequest represents the request to delete a note.
 type NoteDeleteRequest struct {
+	ID uuid.UUID `json:"-"`
```

Comment:
Since the NoteDeleteRequest struct is a representation of the request coming from the frontend, isn't it missleading that you've included the note- & board-id, even tho they are extracted from the URL and are not contained within the request body?